.github/workflows/main.yaml

```diff
@@ -38,27 +41,28 @@ jobs:
       - name: Build
         run: |
           scripts/alpine-setup.sh
-          GOOS=darwin GOARCH=amd64 make all
-          GOOS=linux GOARCH=amd64 make all
+          GOOS=${{ matrix.os }} GOARCH=amd64 make all
           make changelog
+          mv release-artifacts release-artifacts-${{ matrix.os }}
         shell: sh
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Archive release artifacts
         uses: actions/upload-artifact@v1
         with:
-          name: release-artifacts
-          path: release-artifacts
-  release:
-    name: Release
-    needs: build
+          name: release-artifacts-${{ matrix.os }}
+          path: release-artifacts-${{ matrix.os }}
```

Comment:
Related to the above - if we remove the var here, I think we don't need the `mv` above